app/uk/gov/hmrc/cgtpropertydisposals/service/returns/transformers/ReturnTransformerService.scala

```diff
@@ -159,6 +161,18 @@ class ReturnTransformerServiceImpl @Inject() (
       AmountInPence.fromPounds(singleDisposalDetails.acquisitionFees)
     )
 
+  private def constructInitialGainAnswers(
+    singleDisposalDetails: SingleDisposalDetails
+  ): Option[AmountInPenceWithSource] = {
+    val amount: Option[AmountInPence] =
+      (singleDisposalDetails.initialLoss, singleDisposalDetails.initialGain) match {
+        case (Some(loss), _) => if (loss > 0) Some(AmountInPence.fromPounds(-loss)) else Some(AmountInPence(0L))
```

Comment:
we'll always get positive amounts from DES since we can only send down positive numbers

Reply:
are you saying this about the `if (loss > 0)` or the actual coercion to a negative when there's Some(loss)? We can do without the `if (loss > 0)` stuff because it will return zero anyway...

Reply:
yeah, I was on about the `loss > 0` part - we should be able to check the value as is and just stick a minus in front of it

Reply:
I removed the if greater than zero checks